docs/conf.py

```diff
@@ -76,13 +67,42 @@
     'sphinx.ext.autodoc',
     'sphinx.ext.napoleon',
     'sphinx.ext.mathjax',
-    'recommonmark',
-    'sphinx_markdown_tables',
```

Comment:
Can you show us a screenshot for table rendering after this change？

Reply:
some tables are broken ...
<img width="1712" alt="image" src="https://user-images.githubusercontent.com/26535726/229412522-7efc347e-c33a-4462-aab0-bc7f160c7585.png">

Reply:
I would leave my -1, if tables are malformed

Reply:
struggled some hours, no idea why it broken and how to fix it, reverting then